teensy_audio_octo/include/makeColor.h

```diff
@@ -0,0 +1,4 @@
+unsigned int h2rgb(unsigned int v1, unsigned int v2, unsigned int hue);
```

Comment:
should it be exposed publicly?
I think it's only used inside `makeColor.cpp` right?

I even think it's only called from `makeColor(h, s, l)` which is unused itself, so maybe get rid of it altogether.